content/docs/command-reference/fetch.md

```diff
@@ -106,6 +107,19 @@ The `dvc remote` used is determined in order, based on
   workspace. This downloads tracked data for the entire commit history of the
   project.
 
+- `--max-size <bytes>` - fetch data files/directories that are each below
+  specified size (bytes).
+
+  <admon type="info">
```

Comment:
sorry, I meant more like in the paragraph itself, to make it clear that it's more of a Caveat that one rarely needs to know. I'm kinda trying to balance its existence and make it as less visible as possible. I can take it over if needed. 

It's also minor I think.

Reply:
Not sure I follow. What paragraph?

Reply:
Oh, I think I get it now.